app/assets/javascripts/client/about/AboutSidebarCtrl.js

```diff
@@ -0,0 +1,68 @@
+(function() {
+  'use strict';
+
+  angular.module('PDRClient')
+      .controller('AboutSidebarCtrl', AboutSidebarCtrl);
+
+  AboutSidebarCtrl.$inject = [
+    '$scope',
+    '$rootScope',
+    '$element',
+    '$location',
+    '$timeout',
+    '$modal'
+  ];
+
+  function AboutSidebarCtrl($scope, $rootScope, $element, $location, $timeout, $modal) {
+    var vm = this;
+
+    new Clipboard("[data-clipboard-target]");
+
+    vm.openAssessmentModal = function() {
+      $modal.open({
```

Comment:
So this applies to all of the modals:

 - What happens when I have more than one type open?  Is it the case that the other modal is guaranteed to be destroyed?
 - What happens when an unauthenticated user invokes these?  Should we notify them that they have to sign in to have the full functionality of it?